Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -77,14 +77,11 @@ final class PersonCell: UICollectionViewCell {
     }
     
     func configure(with person: Person) {
-        let df = DateFormatter()
-        df.dateFormat = "dd.MM.yyyy"
-        
         profileImageView.image = UIImage(named: "man-in-red-jacket-1681010")
         // ^Update to default place holder image when resource is added.
         // TO-DO: Use Kilo's ImageDownloader here to DL image
         nameLabel.text = person.fullName.uppercased()
-        dateOfIncidentLabel.text = df.string(from: person.date) 
+        dateOfIncidentLabel.text = DateFormatterService().localizedString(date: person.date, dateStyle: .short)
```

Comment:
@hybridcattt what about making services singletons. 

`DateFormatterService.shared.localizedString(date: person.date, dateStyle: .short)` 

So we are not creating instances of services in multiple places. The dateFormatter is only being used here for now but could potentially grow.

Reply:
Very good point about singleton, just I think it should rather be
`DateFormatterService.shared.dateOfIncidentString(date)` , because the goal is to have consistent date styles across the app while keeping the code duplication low, and not so much that default API is not enough. 
The internal implementation of that kind of method can even use `localizedString...` indeed like you mentioned above 👍

Reply:
@hybridcattt @Mackis when first implementing the DateFormatter service I created it as a singleton, but got some push back that we should use it in Service.  I agree with @Kilo-Loco  that we probably should be passing it in as an arg to PersonCell.

Reply:
Interesting. The `Person` model doesn't have a `date` object? Where is this coming from?

Reply:
We are also going with dependency injection, so we'd need our cells and whatnot to work with `ServiceReferring`.